modular_skyrat/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -0,0 +1,36 @@
+/mob/living/simple_animal/hostile/asteroid/goliath/zombie
+	name = "zombie"
+	desc = "Why is he so blocky?"
```

Comment:
if its a guy then maybe gender = MALE it

Reply:
i will yes